presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -46,6 +46,7 @@
     private boolean partitionVersioningEnabled;
     private MetastoreCacheScope metastoreCacheScope = MetastoreCacheScope.ALL;
     private boolean metastoreImpersonationEnabled;
+    private boolean partitionCacheVerificationEnabled;
```

Comment:
Shall we make it a 0 - 100 integer to sample how much traffic we do validation? Check `orcWriterValidationPercentage` as an example

Reply:
Done.